src/components/Button/Button.module.css

```diff
@@ -0,0 +1,100 @@
+/* ------------------ BUTTONS ------------------ */
```

Comment:
- Remove all margins. For generic components always leave positional decisions to the user. In this case if you place margins you would have problem if you wanted to use same component on some specific place with smaller margin so it's best to not include that kind of data within generic components